help/en/releasenotes/current-draft-note.shtml

```diff
@@ -639,7 +639,11 @@
    <h3>Web Server</h3>
         <a id="server" name="server"></a>
         <ul>
-            <li></li>
+            <li>Added a new "web/idtags_examples" directory with an example
+                of  as icons, which move to different places on the webpage depending on their most recent report. 
+                The example shows cars being switched in a simple yard, 
+                the principles shown could be used to show locomotives or train movements.
+                See the icluded PDF file for more information.</li>
```

Comment:
"icluded" should be "included"

Reply:
I made all the suggested changes to the release note.  Thank you!